docs/index.html

```diff
@@ -468,9 +468,9 @@ <h4>Expected Contexts</h4>
 
 <h3 id="person-snippet-singular-block">Person Snippet - Singular - Block</h3><a class="permalink" href="#person-snippet-singular-block">permalink</a>
 
-<h4>Expected Contexts</h3>
+<h4>Expected Contexts</h4>
 <ul>
-  <li><span class="distinct">article.persons > ul > li > article.person</span> </li>
+  <li><span class="distinct">article.persons ul > li > article.post</span> </li>
```

Comment:
This change is unrelated to the Issue in question, please remove.

Reply:
Will do that. It was a typo error

Reply:
Hi @possumbilities 
The typo error have been corrected